src/extractor/restriction_parser.cpp

```diff
@@ -126,6 +126,25 @@ RestrictionParser::TryParse(const osmium::Relation &relation) const
         {
             return {};
         }
+
+        if (parse_conditionals)
+        {
+            // Parse condition and add independent value/condition pairs
+            const auto &parsed = osrm::util::ParseConditionalRestrictions(value);
+
+            if (parsed.empty())
+                continue;
+
+            for (const auto &p : parsed)
+            {
+                std::vector<util::OpeningHours> hours = util::ParseOpeningHours(p.condition);
+                // found unrecognized condition, continue
+                if (hours.empty())
+                    return {};
```

Comment:
The same, if we found something unexpected we just return, should it be just ignoring instead?